src/reflect/scala/reflect/internal/Types.scala

```diff
@@ -2490,7 +2490,7 @@ trait Types
     //TODO this may be generalised so that the only constraint is dependencies are acyclic
     def approximate: MethodType = MethodType(params, resultApprox)
 
-    override def safeToString = paramString(this) + resultType
+    override def safeToString = paramString(this) + ": " + resultType
```

Comment:
I would recommend restricting the changes for this PR to the interpreter, as this method in reflect should stay the same.